pkgs/applications/science/electronics/pcb/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, pkgconfig, gtk, bison, intltool, flex, netpbm, imagemagick, dbus, xlibsWrapper, mesa, desktop_file_utils,  shared_mime_info, tcl, tk, gnome, pangox_compat, gd, xorg }:
+
+stdenv.mkDerivation rec {
+  name = "pcb-${version}";
+  version = "20140316";
+
+  src = fetchurl {
+    url = "http://ftp.geda-project.org/pcb/pcb-20140316/${name}.tar.gz";
+    sha256 = "0l6944hq79qsyp60i5ai02xwyp8l47q7xdm3js0jfkpf72ag7i42";
+  };
+
+  buildInputs = [ pkgconfig gtk bison intltool flex netpbm imagemagick dbus xlibsWrapper mesa desktop_file_utils tcl shared_mime_info tk gnome.gtkglext pangox_compat gd xorg.libXmu ];
+
+  meta = with stdenv.lib; {
+    description = "PCB: Printed Circuit Board editor";
+    homepage = http://pcb.geda-project.org/;
+    maintainers = with maintainers; [ mog ];
+    platforms = platforms.linux;
+    license = licenses.gpl2;
+    priority = 10;
```

Comment:
I don't think we should ship `/share/applications/mimeinfo.cache` at all. This file is actually generated by `update-desktop-database`. This would make the priority hack obsolete.